R/misc.R

```diff
@@ -0,0 +1,25 @@
+#' Pass command lines to a shell
+#'
+#' Cross-platform function to pass a command to the shell, using either
+#' \code{\link[base]{system}} or
+#' (Windows-only) \code{\link[base]{shell}}, depending on the operating system.
+#'
+#' @param commandstring String.
+#' One or multiple commandlines as one would enter in the shell.
```

Comment:
Maybe use the same description as in `base::shell`: "the system command to be invoked, as a string.?
I think "multiple" is not correct.

Reply:
Multiple is in the sense of 'a string of multiple commands'.

Example (or with `shell()`):

```r
execshell("git status
           git log --oneline -n 10")
```

Should I rephrase better instead?

Reply:
OK, I didn't know that something like that would be possible. But I understood multiple to mean `c("first command","second command")`.  So something like 'The command to be invoked, as a string. It is also possible to pass multiple commands at once, as long as they are passed as a single string'?

Reply:
Suggestion 068a83a: 
```
#' @param commandstring
#' The system command to be invoked, as a string.
#' Multiple commands can be combined in this single string, e.g. with a
#' multiline string.
```

As a sidenote: I think the `commandstring` argument name was chosen earlier for contrast with `rgrass7::execGRASS()`, where an argument `cmd` means a single command name (without whitespaces), as in `system2()`.

- Recently a function `rgrass7::stringexecGRASS()` was added, and it has the `string` argument with more flexibility (but will not accept multiple commands as it is still a wrapper around `execGRASS()`).
- `system()` has `command`, which is actually very flexible as you could see.

So feel free to change the name `commandstring` in one or the other direction (`string`/ `command`).

Reply:
Good description. I will maybe change it to `command` later.